metis-core/metis-core-rest/src/main/java/eu/europeana/metis/core/rest/exception/RestResponseExceptionHandler.java

```diff
@@ -103,4 +104,29 @@ public StructuredExceptionWrapper handleMessageNotReadable(Exception exception,
     return new StructuredExceptionWrapper(
         "Request not readable.\n" + exception.getMessage());
   }
+
+  /**
+   * Handler for specific classes to overwrite behaviour
+   * @param exception the exception thrown
+   * @param response the response that should be updated
+   * @return {@link StructuredExceptionWrapper} a json friendly class that contains the error message for the client
+   */
+  @ExceptionHandler(value = MissingRequestHeaderException.class)
+  @ResponseBody
+  public StructuredExceptionWrapper handleMissingRequestHeaderException(Exception exception,
+      HttpServletResponse response) {
+
+    String header = ((MissingRequestHeaderException) exception).getHeaderName();
+
+    StructuredExceptionWrapper output = new StructuredExceptionWrapper(exception.getMessage());
+    response.setStatus(HttpStatus.BAD_REQUEST.value());
+
+    if(header.equalsIgnoreCase("Authorization")) {
+      response.setStatus(HttpStatus.UNAUTHORIZED.value());
+      output = new StructuredExceptionWrapper(
+          "Authorization token is missing.");
```

Comment:
It's not the token that is missing but the whole header is not provided, perhaps write something along the lines "Authorization header is missing in the request"?

Reply:
That is more specific and better, will write that instead